docs/api.md

```diff
@@ -1,9 +1,33 @@
 # Editor.js API
 
 
-Blocks have access to the public methods provided by Editor.js API Module. Plugin and Tune Developers
+Tools have access to the public methods provided by Editor.js API Module. Plugin and Tune Developers
 can use Editor\`s API as they want.
 
+## Block API
+
+API for certain Block methods and properties. You can access it through `editor.api.block.getBlockByIndex` method or get it form `block` property of [Tool constructor](../types/tools/block-tool.d.ts) argument.
+
+`name: string` — Block's Tool name
+
+`config: ToolConfig` — Tool config passed on Editor initialization
+
+`holder: HTMLElement` — HTML Element that wraps Tool's HTML content
+
+`isEmpty: boolean` — `true` if Block has any editable content
+
+`selected: boolean` - `true` if Block is selected with Cross-Block Selection
+
+`set stretched(state: boolean)` — set Block's stretch state
+
+`stretched: boolean` — `true` if Block is stretched
+
+`call(methodName: string, param?: object): void` — method to call Tool's methods with checks and error handlers under-the-hood
+
+`save(): Promise<void|SavedData>` — returns data saved from current Block's state, including Tool name and saving exec time
```

Comment:
- what is the difference between `SavedData` and `BlockToolData`? 
- ` saving exec time` - we include it only for common saving, not for every block.

Reply:
`BlockToolData` is an object, returned from `save` method of Tool instance. `Block` instance `save` wraps `BlockToolData` with `SavedData` object.

`BlockToolData`
```
{
  "text": "Some text"
}
``` 

`SavedData`
```
{
  "tool": "paragraph",
  "data": {
    "text": "Some text"
  },
  time: 0.1923
}
```

General `editor.save` method uses time from `SavedData` from each Block to calculate final exec time.

I agree that we have some inconsistency here, but I think it's out of the scope of this feature.